src/EditorFeatures/Core/Implementation/Classification/ClassificationTypeDefinitions.cs

```diff
@@ -306,5 +324,19 @@ internal sealed class ClassificationTypeDefinitions
         [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
         internal readonly ClassificationTypeDefinition XmlLiteralTextTypeDefinition;
         #endregion
+
+        #region Static Symbol
+        [Export]
+        [Name(ClassificationTypeNames.StaticSymbol)]
+        [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
```

Comment:
❓ Why not `Identifier` as the base type?

Reply:
I tired this. We don't want to pull along the foreground and background colors from Identifier and apply them over the more specific classifications.

Reply:
worth a comment to that effect :)